xonsh/xontribs.json

```diff
@@ -43,6 +43,15 @@
   "url": "https://github.com/AstraLuma/xontrib-avox",
   "description": ["Automatic (de)activation of virtual environments as you cd around"]
  },
+ {"name": "back2dir",
+  "package": "xontrib-back2dir",
+  "url": "https://github.com/anki-code/xontrib-back2dir",
+  "description": [
+    "Back to the latest used directory when starting xonsh shell. ",
+    "For example if you changed the directory to ``/work`` and then exit from xonsh ",
+    "and after time you run xonsh again then you'll be in ``/work`` directory at start ",
+    "instead of home directory."]
```

Comment:
"then your next xonsh session will start in the '/work' directory, instead of your home directory."